native/packages/react-native-bpk-component-animate-height/readme.md

```diff
@@ -0,0 +1,66 @@
+# react-native-bpk-component-animate-height
+
+> Backpack React Native animate height component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-animate-height --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import BpkAnimateHeight from 'react-native-bpk-component-animate-height';
+import BpkButton from 'react-native-bpk-component-button';
+import BpkText from 'react-native-bpk-component-text';
+import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: TOKENS.spacingBase,
+  },
+  animateHeight: {
+    marginBottom: TOKENS.spacingBase,
+  }
+});
+
+export default class App extends Component {
+  state = {
```

Comment:
This is proposed syntax (see https://github.com/tc39/proposal-class-fields). I reckon we should stick to proposals that have actually made the spec (stage 4).

See https://github.com/tc39/proposals for more info.